main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"io"
+	"log"
+	"net/http"
+)
+
+func main() {
+	helloHandler := func(w http.ResponseWriter, req *http.Request) {
+		io.WriteString(w, "Hello! This is orwelly-api\n")
+	}
+
+	http.HandleFunc("/hello", helloHandler)
+    log.Println("Listing for requests at http://localhost:8000/hello")
```

Comment:
8080 is the classical number for this, but it's ok.

Reply:
Thanks, I'll consider it next time!